backend/src/events/match/functions/endMatch.ts

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Adicionar importação do tipo `Player`

```suggestion
import Player from "../../../interfaces/Player";

```